### PR TITLE
Ticket 3

### PR DESCRIPTION
**Pull request by smriteemanandhar**
fixed overlay of username

### CODE REVIEW

File: src/components/Login.css [hunk 1]
```diff
@@ -4,7 +4,8 @@
   left: 50%;
   transform: translate(-50%, -50%);
   width: 400px;
-  background: white;
+  background: wh
+  ite;
```
Comment (sanjayguwaju) [inline src/components/Login.css]: it should  be in one line like background: white; but its better practice to use rgb for example :  use #fff for white and #000 for black.

File: src/components/Login.jsx [hunk 2]
```diff
@@ -36,12 +36,12 @@ function Login() {
             <h1 className = "heading">Login</h1>
             <div className = "form">
             <div className="txt_field">
-                <input type="text" required placeholder="Username" onChange={e => setUsername(e.target.value)}/>
+                <input type="text" required onChange={e => setUsername(e.target.value)}/>
```
Comment (sanjayguwaju) [inline src/components/Login.jsx]: input should have placeholder for good user experience.

File: src/components/Login.jsx [hunk 3]
```diff
@@ -36,12 +36,12 @@ function Login() {
...
                 <span></span>
                 <label>Username</label>
             </div>
             <div className="txt_field">
-                <input type="password" placeholder="Password" onChange={e => setPassword(e.target.value)}/>
+                <input type="password" onChange={e => setPassword(e.target.value)}/>
```
Comment (sanjayguwaju) [inline src/components/Login.jsx]: input should have placeholder for good user experience.

**Review by sanjayguwaju — CHANGES_REQUESTED** [review]
please view the comments and make the changes.

File: src/components/Login.css (hunk 1 above)
Comment (smriteemanandhar) [reply src/components/Login.css]: done.

File: src/components/Login.jsx (hunk 2 above)
Comment (smriteemanandhar) [reply src/components/Login.jsx]: done

File: src/components/Login.jsx (hunk 3 above)
Comment (smriteemanandhar) [reply src/components/Login.jsx]: completed

**Pull request closed without merging**